integration-tests/bats/db-revision-specifiers.bats

```diff
@@ -3,6 +3,7 @@ load $BATS_TEST_DIRNAME/helper/common.bash
 
 setup() {
     setup_common
+    export DOLT_DISABLE_DATABASE_RENAMING="false"
```

Comment:
The value of `false` is a little misleading, because we don't ever actually check the value, just whether the env var is set or not. Instead, just use `true` or `1` for settings where we're only checking the existence of the setting, and not checking the actual value. 

It's also a bit odd to see `disable=false` – avoid double negatives like that whenever you see them. They easily lead to confusion in code and in prose.